EscapeVR/src/main/java/nl/tudelft/kroket/event/EventListener.java

```diff
@@ -6,12 +6,12 @@
 
 public interface EventListener extends Event {
 
-	@Override
-	public default void handleEvent(EventObject e) {
-		
-		if (e instanceof InteractionEvent) {
-			System.out.println("event received: " + e.toString());
-		}
-	}
+//	@Override
```

Comment:
I did want to keep a default entry. I would keep it there if jMonkeyEngine wouldn't complain about it. I guess it can be removed.